doc/cookbook/source/examples/multiclass_classifier/cartree.rst

```diff
@@ -0,0 +1,47 @@
+==================================
+Classification And Regression Tree
+==================================
+
+Decision tree learning uses a decision tree as a predictive model which maps observations about an item to conclusions about the item's target value.
+
+Decision trees are mostly used as the following two types:
+
+- Classification tree, where the predicted outcome is the class to which the data belongs.
+- Regression tree, where predicted outcome can be considered a real number.
+
+Classification And Regression Tree (CART) algorithm is an umbrella method that can be applied to generate both classification tree and regression tree.
+
+In this example, we showed how to apply CART algorithm to multi-class dataset and predict the labels with classification tree.
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CMulticlassLabels` as
+
+.. sgexample:: cartree.sg:create_features
+
+We set the type of each predictive attribute (true for nominal, false for ordinal/continuous)
```

Comment:
These words you can also use above when you talk about regression/classification values